public/gallery.html

```diff
@@ -106,12 +132,180 @@
 			<button class="bg-black text-white text-lg ring-2 py-1 px-4 rounded-lg self-center" type="submit"><strong>Apply</strong></button>
 		</div>
 	</div>
-<!-- End filters container -->
 </div>
+<!-- End filters container -->
+
 <!-- Menu end -->
-<div>
-	<img  src="Assets/images/gallery/percare_gallery_hero.png" alt="Animals posing in a art gallery">
+
+<!-- Grid section -->
+<div id="backgroundImg">
+
+<!-- Grid container start -->
+	<div class="w-full h-full grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4" id="imgContainer">
+		<!-- Grid box container -->
+		<div class="imageHoverContainer relative">
+			<img src="Assets/images/gallery/gallery_grid/dog-1.png" alt="dog portrait">
+			<div class="statsBox flex flex-col justify-center items-center  absolute bottom-0 inset-x-0 bg-purple-400 h-24 bg-opacity-90 opacity-0">
+				<div class="text-15 font-bold text-blue-800">Gender: male</div>
+				<div class="text-15 font-bold text-blue-800">Rating: 4☆</div>
+			</div>
+		</div>
+
+		<div class="imageHoverContainer relative">
+			<img src="Assets/images/gallery/gallery_grid/cat-1.png" alt="dog portrait">
+			<div class="statsBox flex flex-col justify-center items-center  absolute bottom-0 inset-x-0 bg-purple-400 h-24 bg-opacity-90 opacity-0">
+				<div class="text-15 font-bold text-blue-800">Gender: male</div>
+				<div class="text-15 font-bold text-blue-800">Rating: 3☆</div>
+			</div>
+		</div>
+
+		<div class="imageHoverContainer relative">
```

Comment:
Again this name is too tight with your css intentions. 

So for a mobile device, what does the class image hover do? There is no hover on mobile. 

Also in one place you are using "image" and in another it's "img"